scripts/smoke_tests/commands_test.ps1

```diff
@@ -6,16 +6,31 @@
 # --------------------------------------------------------------------------------------------
 
 # Setup
+if (!$args[0] -Or !$args[1]) {
+    Write-Error -Message "Error: Both resource group and central app id arguments are mandatory to run the script." -Category InvalidArgument
+    exit
```

Comment:
This should exit with a non-zero integer.

Reply:
Makes sense